src/Symfony/Component/Notifier/Bridge/SmsBiuras/SmsBiurasTransportFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\SmsBiuras;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class SmsBiurasTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return SmsBiurasTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+
+        if ('smsbiuras' !== $scheme) {
+            throw new UnsupportedSchemeException($dsn, 'smsbiuras', $this->getSupportedSchemes());
+        }
+
+        $uid = $this->getUser($dsn);
+        $token = $this->getPassword($dsn);
+        $from = $dsn->getRequiredOption('from');
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $port = $dsn->getPort();
+        $test = $dsn->getRequiredOption('test');
```

Comment:
Please move below the from and make it optional by using:

$test = $dsn->getOption('test', false)

Reply:
Done